README.md

```diff
@@ -1,11 +1,13 @@
-# nRF Connect
+# nRF Connect for Desktop
 
 [![Build Status](https://dev.azure.com/NordicSemiconductor/Wayland/_apis/build/status/pc-nrfconnect-core?branchName=master)](https://dev.azure.com/NordicSemiconductor/Wayland/_build/latest?definitionId=8&branchName=master)
 [![License](https://img.shields.io/badge/license-Modified%20BSD%20License-blue.svg)](LICENSE)
 
-nRF Connect is a cross-platform framework for creating desktop apps for use with development kits or dongles from Nordic Semiconductor. It provides a common foundation for creating apps that communicate with the devices over serial port. The framework comes with a skeleton that has standard UI components for selecting devices, navigation menus, logging, etc. Apps can decorate the standard components, create new components, and use built-in libraries in order to create end-user tools.
+nRF Connect for Desktop is a cross-platform framework for desktop apps for development kits or dongles from Nordic Semiconductor. The framework provides a launcher and the general layout for selecting devices, navigation menus, logging, etc. Apps can decorate the standard components and use built-in libraries in order to create end-user tools.
 
-nRF Connect supports the following devices:
+# Supported Platforms
+
+nRF Connect for Desktop supports the following devices:
```

Comment:
I suggest we don't list in detail the different kits that are supported. How about something like:

nRF Connect for Desktop aims to support all official Nordic Semiconductor development kits and dongles including these device families:
* nRF51
* nRF52
* nRF91

Note that while the the framework has support for these device families, the support in individual apps may vary.